packages/react-ui-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rentpath/react-ui-core",
-  "version": "15.2.1",
+  "version": "15.2.2",
```

Comment:
This doesn't bump the datepicker. You'll need to add the datepicker commitish on this line: https://github.com/rentpath/react-ui/pull/383/files#diff-381a8d04c2e36ba268c0723c3a9ce2c5L56

Reply:
@tadjohnston That's the part I was confused about--it's just a reference to the github repo, so how to I show that it's a new version?

Reply:
👍 updated

Reply:
You still don't need this line 😄 . Publishing does this automatically for you.

Reply:
...oh. Gotcha. ha